### PR TITLE
UI fixes : button hover state, responsiveness, preloading SVG

### PR DESCRIPTION
**Pull request by julien51**

### CODE REVIEW

File: unlock-app/src/pages/_document.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default class MyDocument extends Document {
     return (
       <html>
         <Head>
+          <meta name='viewport' content='width=device-width' />
```
Comment (benwerd) [inline unlock-app/src/pages/_document.js]:
```suggestion
          <meta name='viewport' content='width=device-width, initial-scale=1' />
```

File: unlock-app/src/pages/index.js [hunk 2]
```diff
@@ -48,6 +48,7 @@ export const Home = () => (
     <Section>
       <CallToAction>Check out our open source code on <a href="https://github.com/unlock-protocol/unlock">GitHub</a>, come work <a href="/jobs">with us</a> or simply <a href="mailto:hello@unlock-protocol.com">get in touch</a>.</CallToAction>
     </Section>
+
```
Comment (benwerd) [inline unlock-app/src/pages/index.js]: Remove this blank line?

File: unlock-app/src/pages/_document.js (hunk 1 above)
Comment (julien51) [reply unlock-app/src/pages/_document.js]: Thanks!

**Comment by julien51** [comment]
Ok, updated snapshots too!

**Pull request merged**